packages/php/src/Metrics.php

```diff
@@ -221,7 +228,7 @@ private function processRequest(Request $request): array
     private function processResponse($response): array
     {
         if ($response instanceof JsonResponse) {
-            $body = $response->getData();
+            $body = $response->getData(true);
```

Comment:
Made this change because by default `JsonResponse::getData()` will return a `stdClass`, and since we're accessing `$body` as an associative array in the `excludeDataFromBlacklist` and `excludeDataNotInWhitelist` methods those would fail.

Reason this didn't get caught sooner was that I was mocking out the return of `getData` in tests instead of letting that handle its own business.